src/utils.ts

```diff
@@ -61,3 +61,7 @@ export function filename(path: string) {
 function _compareAliases(a: string, b: string) {
   return b.split("/").length - a.split("/").length;
 }
+
+export function _isRootPath(path: string) {
+  return path === "/" || /^\/[a-z]:$/i.test(path);
```

Comment:
MAybe call it _isAbsolutePath? Also windows drive would be nice if we explicitly check for `a-zA-Z` for drive path casing.

Reply:
Thanks. That is a very nice suggestion, I have optimized the code.